### PR TITLE
Use generic run tests

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

**Comment by zmiklank** [comment]
[test-all]

File: test/run_test [hunk 1]
```diff
@@ -36,14 +36,12 @@ test -n "${IMAGE_NAME-}" || false 'make sure $IMAGE_NAME is defined'
 test -n "${VERSION-}" || false 'make sure $VERSION is defined'
 test -n "${OS-}" || false 'make sure $OS is defined'
 
-CIDFILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
+CID_FILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
```
Comment (phracek) [inline test/run_test:39]: Why this is needed?

Comment (phracek) [reply test/run_test:39]: This does not break functionality. And only cosmetic issue does not make sense to me.

**Review by phracek — CHANGES_REQUESTED** [review]
Please look at my comments. I don't think, renaming variable is needed.

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
RHEL-9 Unsubscribed host is removed by #466

File: test/run_test [hunk 2]
```diff
@@ -735,8 +706,6 @@ ${mount_opts}
   # need this to wait for the container to start up
   PGUSER=user     PASS=password           test_connection "$container_name"
   PGUSER=backuser PASS=pass     DB=backup test_connection "$container_name"
-
-  docker stop "$(get_cid $container_name)" >/dev/null
```
Comment (phracek) [inline test/run_test:739]: Why `docker stop` is removed from here?

**Review by phracek — CHANGES_REQUESTED** [review]

File: test/run_test (hunk 2 above)
Comment (zmiklank) [reply test/run_test:739]: Because it is done in `ct_cleanup`.

**Review by phracek — APPROVED** [review]
LGTM. Thanks for your responses. Tests are passing.

**Pull request merged**